### ISSUE TITLE
Register/Login service password error

### ISSUE THREAD

**Issue by scotthooker** [signalpoint/DrupalGap#217]
There is a bug with the register service that attempts to auto log the user in after a successful registration. I see the user being created but then get an alert that the password is wrong or incorrect when it attempts to login .

**Comment by scotthooker** [comment]
At some point the user password on the account create submit is being set to '1' I'm not sure if this is in drupal or drupalgap. I think the former.

**Comment by scotthooker** [comment]
Its a bug in the services module. I've made an issue on d.o https://drupal.org/node/2198005

**Comment by signalpoint** [comment]
The user register service (in drupalgap) only attempts to login the user after registration if e-mail verification is disabled, and no admin approval is required. It looks like you tracked down the bug with the Services module, we'll wait to see what they say.

I think the Services module used to automatically login a user after registration, but I'm pretty sure they took that out. I don't remember why, or if that is even true. But either way, I had to add that manual login feature to the user register form submit.

I just noticed there is one alert() left in the DG user register form submit, that needs to be changed to drupalgap_alert().
